tests/test_executors.py

```diff
@@ -8,7 +8,7 @@
 from my_executors import *
```

Comment:
Preferably, `*` shouldn't be used. It's confusing and ambiguous.

One way would be to write this as,

```python
import my_executors as executors
executors.my_class()
executors.my_func()
```